syntax = "proto3";
package pub_subscriber;




import "model/notify/pub_subscriber.proto";

/*
PubSubscriberList请求
*/
message PubSubscriberListRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页大小
    */
    int32 page_size = 2 ;
}

/*
PubSubscriberList返回
*/
message PubSubscriberListResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated notify.PubSubscriber list = 4 ;
}

/*
PubSubscriberListApi返回
*/
message PubSubscriberListResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    pub_subscriber.PubSubscriberListResponse data = 4 ;
}

